File: Microsoft .NET/.NET Framework 4/ASP.NET/VB.NET/MVC Web Reporting Sample/MVC Web Reporting Sample/ViewModels/Html5ViewerModel.vb
Imports combit.ListLabel25.Web
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

Namespace WebReporting.ViewModels
    Public Class Html5ViewerModel

        Public Sub New()
            ViewerOptions = New Html5ViewerOptions()
        End Sub

        Public Property ReportName() As String
            Get
                Return m_ReportName
            End Get
            Set
                m_ReportName = Value
            End Set
        End Property
        Private m_ReportName As String

        Public Property ViewerOptions() As Html5ViewerOptions
            Get
                Return m_ViewerOptions
            End Get
            Set
                m_ViewerOptions = Value
            End Set
        End Property
        Private m_ViewerOptions As Html5ViewerOptions

    End Class
End Namespace

File: Microsoft .NET/.NET Core 3.1/ASP.NET/C#/MVC Web Reporting Sample/MVC Web Reporting Sample/ViewModels/Html5ViewerModel.cs
using combit.ListLabel25.Web;

namespace WebReporting.ViewModels
{
    public class Html5ViewerModel
    {

        public Html5ViewerModel()
        {
            ViewerOptions = new Html5ViewerOptions();
			ViewerOptions.ShowDetailedErrorMessages = true;
        }

        public string ReportName { get; set; }

        public Html5ViewerOptions ViewerOptions { get; set; }

    }
}
